packages/cxl-ui/scss/cxl-app-layout/_layout.scss

```diff
@@ -23,6 +23,15 @@ $toggle-icon: "lumo:angle-right";
 :host([layout^="2c"]) {
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      *
+      * border-box is used to avoid overflow caused by padding.
+      * 60px is action bar height
+      */
+      box-sizing: border-box;
+      min-height: calc(100% - 60px);
```

Comment:
How come I'm the only one fighting magic numbers @pawelkmpt?

This should be at least a CSS variable, not a hard-coded magic number.

Reply:
> How come I'm the only one fighting magic numbers @pawelkmpt?
> 
> This should be at least a CSS variable, not a hard-coded magic number.

I noticed, investigated, then tested and forgot about moving it to the var.